client/components/banner/index.jsx

```diff
@@ -48,7 +48,8 @@ export class Banner extends Component {
 		event: PropTypes.string,
 		feature: PropTypes.string,
 		href: PropTypes.string,
-		icon: PropTypes.string,
+		icon: PropTypes.oneOfType( [ PropTypes.string, PropTypes.bool ] ),
+		isCompact: PropTypes.bool,
```

Comment:
The Calypso components convention is to name this prop `compact`. See `Card` or `Button` for example.

On the other hand, I see many usages of `isCompact`, too, but not in the base components that should serve as the model for everything else.